src/main/java/io/github/ryuu/adventurecraft/overrides/EntityRenderDispatcher.java

```diff
@@ -67,7 +69,7 @@ private EntityRenderDispatcher() {
         this.o.put(EntityHookshot.class, new RenderHookshot());
         this.o.put(EntityBomb.class, new RenderBomb());
         this.o.put(EntityBat.class, new gv((ko) new ModelBat(), 0.3F));
```

Comment:
I guess you just forgot this `gv`?

Reply:
Oh I did. I did go through it but I missed it. Sometimes my InteliJ doesn't show any errors in a class only after a while :/

Reply:
Should be remapped now